test/utils/helpers.js

```diff
@@ -6,4 +6,6 @@ const isInRange = (amount, range) => {
   return amountF >= parseFloat(from) && amountF <= parseFloat(to)
 }
 
-module.exports = { isAscending, isInRange }
+const removeFilterFields = (searchResults) => searchResults.map(({ scheme, ...rest }) => rest)
```

Comment:
This removeFilterFields() is only removing scheme from the result, Can this be called something like removeSchemeFilter()?

Reply:
That makes sense sure, I will rename it.